src/NHibernate.Test/Linq/QueryLock.cs

```diff
@@ -158,6 +158,7 @@ public void CanSetLockOnBothJoinAndMainComplex()
 		[Test]
 		public void CanSetLockOnLinqPagingQuery()
 		{
+			Assume.That(TestDialect.SupportsSelectForUpdateInSubQueries, Is.True, "Dialect does not support locking in subqueries");
```

Comment:
On my side I was adding a `TestDialect.SupportsSelectForUpdateWithPaging`. The fact that some dialects do implement paging with subqueries is an implementation detail which is not true for all dialects.

Reply:
In fact that property `SupportsSelectForUpdateInSubQueries` has been introduced just for this test case, so it should be renamed to something like `SupportsSelectForUpdateWithPaging`.

Reply:
Done